### ISSUE TITLE
Keep playing music while using other apps

### ISSUE THREAD

**Issue by debaba** [Rello/audioplayer#269]
Hi, nice Player.

Would be nice, if the player kept on playing albums or playlists when changing to another app in NextCloud.

Thanks

**Comment by Rello** [comment]
Hello,
the problem is, that NC is reloading the whole window on every major navigation. due to this, every script execution is being reset when navigating to another app.

In case e.g. the header would remain, the player could be attached there - but this is not how NC is designed.
the only way to do this is to open a second Tab/Window in parallel

**Comment by debaba** [comment]
Right the way i did until now, but sometimes i forget, then..... you know, just for convenience.
Is there no way to keep a hidden window or somewhat else?

**Comment by xraMsamohT** [comment]
Duplicate of #146
